commands/push.js

```diff
@@ -1,77 +1,95 @@
-'use strict';
+const cli = require('heroku-cli-util')
+const Sanbashi = require('../lib/sanbashi')
 
-const cli = require('heroku-cli-util');
-const co = require('co');
-const child = require('child_process');
+let usage = `
+    ${cli.color.bold.underline.magenta('Usage:')}
+    ${ cli.color.cmd('heroku container:push web')}         	           # Pushes Dockerfile in current directory to web process type
+    ${ cli.color.cmd('heroku container:push web worker --recursive')}     # Pushes Dockerfile.web and Dockerfile.worker found in current & subdirectories
```

Comment:
soon(-ish) we plan to have a format for examples since we use them so much: https://github.com/heroku/cli-engine/issues/190